rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -189,6 +190,124 @@ QIcon main_window::GetAppIcon()
 	return m_appIcon;
 }
 
+void main_window::InstallDiscExtra(const std::string path)
```

Comment:
Hi. I'm new here so probably it's a dumb question but usually I'm using reference to string (const std::string& path) instead of copying it. Why it's done like this here and later?
Below I see another method that uses reference (void RequestGlobalStylesheetChange(const QString& sheetFilePath)).